### PR TITLE
Introducing the Recognizers Store service API

### PR DESCRIPTION
**Pull request by eladiw**
This protobuf change introduce the support for the RecognizersStore service, which is responsible to handle the recognizers with the underlying storage.

This change is a prerequisite to the change that will be added in the Presidio main repo.

Note to the reviewers, the .go and .py are auto-generated, pay most attention to the .proto file

### CODE REVIEW

File: src/recognizers_store.proto [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+package types;
+
+// PatternRecognizer is the object stored in the store
+message PatternRecognizer {
+    // The name of the recognizer, unique
+    string name = 1;
+    // The Regex used by this recognizer
+    string pattern = 2;
+    // The confidence of the result
+    float strength = 3;
```
Comment (limorl) [inline src/recognizers_store.proto]: I think we should change it to score - will make more sense, cause this is how we use it.
We can change the API for now, and later change the backend (I can do it)

File: src/recognizers_store.proto [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    // The entity name which this recognizer can detect
+    string entity = 4;
+    // The supported language
+    string language = 5;
```
Comment (limorl) [inline src/recognizers_store.proto]: Let's keep all API consistent. either language or languageCode - I prefer language and in the comment write languageCode according to ISO XXX (as specified in the change Itye made)

File: src/recognizers_store.proto [hunk 3]
```diff
@@ -8,11 +8,17 @@ message PatternRecognizer {
     // The Regex used by this recognizer
     string pattern = 2;
     // The confidence of the result
-    float strength = 3;
+    float score = 3;
     // The entity name which this recognizer can detect
     string entity = 4;
-    // The supported language
+    // The supported language code, in ISO-639 format, https://en.wikipedia.org/wiki/ISO_639-1
     string language = 5;
```
Comment (limorl) [inline src/recognizers_store.proto]: @itye-msft please make sure you fix languageCode to language in the API changes you made, to align with @ELad's changes.

File: src/recognizers_store.proto [hunk 4]
```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+package types;
+
+// PatternRecognizer is the object stored in the store
+message PatternRecognizer {
+    // The name of the recognizer, unique
+    string name = 1;
+    // The Regex used by this recognizer
+    string regex = 2;
```
Comment (omri374) [inline src/recognizers_store.proto]: The pattern is a message on its own. Its structure is: 
message Pattern:
string name = 1
string regex = 2
float strength = 3
A PatternRecognizer should have a list of such patterns

**Pull request merged**